website/admin/views/admin_user_rooms.py

```diff
@@ -0,0 +1,51 @@
+from flask import redirect, url_for, flash, render_template
+
+import data_interface
+from admin import admin_site
+from website.forms import AddNewRoomForm
```

Comment:
There is no module named website. The root directory of the website component is already that directory. If you want to keep the forms.py file there, you should import it as:
from forms import ...